### ISSUE TITLE
TrulyHandsFree library for another platform?

### ISSUE THREAD

**Issue by mykinator** [Sensory/alexa-rpi#7]
Is there a non-commercial (limited trial version) TrulyHandsFree library for another platform besides Raspberry Pi? I looked on the Sensory website and it didn't look like there was any download page or FAQ page addressing this.

**Comment by JHdeVilliers** [comment]
No. This repository is the only trial version of TrulyHandsfree that's currently publicly available.

The TrulyHandsfree SDK runs on various other platforms, but you'll have to go through [Sensory Sales][1] to gain access to these.

[1]: http://www.sensory.com/support/contact/us-sales/
